README.md

```diff
@@ -149,12 +149,18 @@ assert result.column(0) == pyarrow.array([6.0])
 
 ## How to install (from pip)
 
+### Pip
 ```bash
 pip install datafusion
 # or
 python -m pip install datafusion
 ```
 
+### Conda
+```bash
+conda install -c conda-forge datafusion
```

Comment:
Who will update the conda-forge metadata for new releases ?
I.e. would it be possible that a new version of a-d-python is released, e.g. 0.9, and conda still installs 0.8 ? 
If Conda does not fetch this info from PyPi then I expect users to file issues here about this and the a-d-p developers won't be able to help.

Reply:
> It would be the responsibility of the community to update conda-forge at release time. Let me add the logic in the release scripts and document that in this PR.
> 
> 
> 
> For what its worth there is an repo at https://anaconda.org/conda-forge/datafusion and I am trying to find out who owns that now so the proper permissions can be established

I now have permissions for this repo and can add other PMC members